course_info/templates/course_info/partials/people.html

```diff
@@ -38,263 +43,250 @@ <h3 class="breadcrumbs">
         </table>
       </div>
   </div>
+</div>
 
+<div ng-if='courseInstance && initialCourseMembersFetched' class="row add-top-margin">
   <!-- begin add people interface -->
   <!-- begin add person expand-o-widget -->
-  <div role="tabpanel" class="tab-pane active fade in active col-md-12" id="people">
-    <br />
+  <div class="col-md-12" id="people">
     <button id="addPeopleBtn" class="btn btn-primary" type="button"
-            data-toggle="collapse" data-target="#addPeopleDialog"
-            aria-expanded="false" aria-controls="addPeopleDialog">
+            data-toggle="collapse" data-target="#add-people-dialog"
+            aria-expanded="false" aria-controls="add-people-dialog">
        Add People <span class="caret"></span>
     </button>
-    <div class="collapse" id="addPeopleDialog">
-      <div class="well container-fluid col-md-12">
-        <div class="form-group-inline">
-
-          <div class="col-md-6">
-            <div class="form-group">
-              <label class="sr-only" for="emailHUID">
-                Enter Harvard University ID or email address
-              </label>
-              <input class="form-control" id="emailHUID"
-                     placeholder="Enter Harvard University ID or email address"
-                     ng-model="searchTerm" ng-disabled="searchResults.length > 1">
-            </div>
-          </div>
-
-          <div class="col-md-2">
-            <button id="select-role-btn-id"
-                    type="button"
-                    class="btn btn-primary dropdown-toggle"
-                    data-toggle="dropdown"
-                    aria-haspopup="true"
-                    aria-expanded="false"
-                    ng-disabled="searchTerm.length == 0 || searchResults.length > 1"
-                    ng-bind-html="selectedRole.roleName + ' <span class=\'caret\'></span>'">
-            </button>
-            <ul class="dropdown-menu" role="menu"
-                aria-labelledby="select-role-btn-id">
-              <li ng-repeat="role in roles">
-                <a ng-click="selectRole(role)">
-                  {{ role.roleName }}
-                </a>
-              </li>
-            </ul>
-          </div>
+    <div class="well collapse form-horizontal" id="add-people-dialog">
+      <div class="form-group">
+        <div class="col-xs-12">
```

Comment:
If you're nesting columns (`col-xs-12` here, where its great-grandparent is `col-md-12`), isn't there supposed to be a sub-row wrapping this?